bindings/python/.evergreen/test.sh

```diff
@@ -22,6 +24,10 @@ if [ "Windows_NT" = "$OS" ]; then # Magic variable in cygwin
              "C:/python/Python38/python.exe"
              "C:/python/Python39/python.exe"
              "C:/python/Python310/python.exe")
+
+    export CSFLE_PATH=../csfle/bin/mongo_csfle_v1.dll
+    C:/python/Python310/python.exe drivers-evergreen-tools/.evergreen/mongodl.py --component csfle \
+      --version latest --out ../csfle/
```

Comment:
The formatting of each of these three `export CSFLE_PATH` sections is different. This one has a leading newline, the next has a trailing newline, and the last has neither. Can we have some consistency?